content.js

```diff
@@ -56,11 +56,52 @@ function addIndividualIssueButton() {
             })();
 
             button.className = "btn " + emailGitHubIssuesClassName;
-            titleElement.parentNode.insertBefore(button, document.getElementsByClassName("js-issue-title")[0].parentNode.firstChild);
+
+            if (!isPull)
+{
+            var button2 = document.createElement("input");
+            button2.setAttribute("type", "button");
+            button2.setAttribute("value", "Copy as Workitem Attribute");
+            //button2.setAttribute("name", "button2name");
+            var issueNumber = document.getElementsByClassName("gh-header-number")[0].innerHTML.substring(1);
+            var str = '<WorkItem(';
+            var str2= '")>';
+            var str3 = str + issueNumber + ', "' + window.location.href + str2;
+
+            var issueTitle = document.getElementsByClassName("js-issue-title")[0].innerHTML;
+            button2.onclick = (function () {
+                var currentTitle = issueTitle;
+                var currentNumber = issueNumber;
+                var currentIsPull = isPull;
+                return function () {
+                    copyTextToClipboard(str3);
+                };
+            })();
+
+            button2.className = "btn " + emailGitHubIssuesClassName;
+
+            var div = document.createElement("div");
+            div.setAttribute("class", emailGitHubIssuesClassName);
+            div.appendChild(button);
+            div.appendChild(button2);
+            titleElement.parentNode.appendChild(div);
```

Comment:
The "Email PR" button won't show up in PRs now because your div appending only happens in the if (!isPull). Move the ```<div>``` declaration above the ```if``` and add one thing at a time to the div (each step either inside extra conditionals or not) and then at the end add that div (regardless of conditionals).